h2o-core/src/main/java/water/api/ModelsHandler.java

```diff
@@ -256,4 +256,21 @@ public ModelExportV3 exportMojo(int version, ModelExportV3 mexport) {
     return mexport;
   }
 
+  public ModelExportV3 exportModelDetails(int version, ModelExportV3 mexport){
+    Model model = getFromDKV("model_id", mexport.model_id.key());
+    try {
+      URI targetUri = FileUtils.getURI(mexport.dir); // Really file, not dir
+      Persist p = H2O.getPM().getPersistForURI(targetUri);
+      //Output model details to JSON
+      OutputStream os = p.create(targetUri.toString(),mexport.force);
+      os.write(model.writeJSON(new AutoBuffer()).buf());
```

Comment:
Ahh, oki this is crucial - you are exporting internal form of model which can change. You should export model schema, no? Can you pls attache example of exported json?